include/libdnf/base/solver_problems.hpp

```diff
@@ -36,16 +36,17 @@ class SolverProblems {
     /// Provide information about package solver problems in a vector. Each problem can be transformed to string by
     /// package_solver_problem_to_string or all problems to a string by all_package_solver_problems_to_string().
     ///
-    /// @return Vector with structuralized package solver problems
+    /// @return Vector with structuralized package solver problems. The vector conteins solver errors represented
+    /// by a vector with std::pairs that describe each problem
```

Comment:
Sorry, but this is still not clear. What does _"vector contains solver errors represented by a vector"_ mean?

I think it's quite clear the inner vector is a list of errors, but what is in the outer vector?

Also, typo: _conteins_.

Reply:
I updated and I hope that it resolves it.